docs/getting_started.md

```diff
@@ -150,7 +173,7 @@ Running the following command(on the host where you execute `clusterctl` in prev
 ```shell
 $ CONTROL_PLANE_ENDPOINT_IP=10.10.10.10 clusterctl generate cluster byoh-cluster \
     --infrastructure byoh \
-    --kubernetes-version v1.22.0 \
+    --kubernetes-version v1.22.3 \
```

Comment:
Do you also want to update the base image in the makefile to have a consistent experience?
https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/4d149bdfb599cf5e434418e25b2e4b8491d31996/Makefile#L12

Reply:
You are right. I have opened a new issue https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/issues/268 to track this + some other info about `skip-installation` flag that needs to be used when using docker hosts.